inclusive-design/wecount.inclusivedesign.ca
# feat: remove Netlify Large Media integration

## Description

Removes deprecated Netlify Large Media integration.

## Steps to test

1. Check out this branch.
2. Inspect contents of `/src/uploads`.

**Expected behavior:** Media should be present.

## Additional information

You may need to run `git lfs uninstall` once you check out this branch.

## Related issues

Resolves #1555.
